src/gui/Home.py

```diff
@@ -1,6 +1,6 @@
 import streamlit as st
 from src.lib.constants import IMAGE_HOME
-
+st.set_page_config(layout="wide")
```

Comment:
I know, I suggested it... but I don't really like it... everything looks too wide ^^  
unless you feel very strongly about it, I'd remove it again